src/test/features/response/routes/eligibility/send-your-resonse-by-email.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai'
```

Comment:
Please correct file name, it should be `send-your-response-by-email.ts` instead of `send-your-resonse-by-email.ts`.